pages/blog/index.vue

```diff
@@ -46,10 +46,28 @@ if (data.value.entry && import.meta.prerender) {
   }
 }
 
+// DATA
 const page = ref(_get(data.value, 'entry', {}))
 const featuredArticles = page.value.ftvaFeaturedArticles
 const articleList = ref(_get(data.value, 'entries', {}))
-console.log(data.value)
+console.log('Data: ', data.value)
+
+const currentPage = ref(1)
+const documentsPerPage = 10
+
+// TEST
+const { paginatedArticlesQuery } = useArticlesListSearch()
+
+onMounted(async () => {
+  const esOutput = await paginatedArticlesQuery(
+    currentPage.value,
+    documentsPerPage,
+    'startDate',
```

Comment:
This sample call to the composable passes date argument as `startdate`, I think it needs to be postDate https://github.com/UCLALibrary/ftva-website-nuxt/blob/cee9b453ee5c7277ce2435705cd6a1eac97ec2e6/gql/queries/FTVAArticleDetail.gql#L9C5-L9C13